org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/services/XMLHighlighting.java

```diff
@@ -91,12 +91,12 @@ public XMLHighlighting(XMLExtensionsRegistry extensionsRegistry) {
 		return Collections.emptyList();
 	}
 
-	private static boolean doesTagCoverPosition(Range startTagRange, Range endTagRange, Position position) {
+	public static boolean doesTagCoverPosition(Range startTagRange, Range endTagRange, Position position) {

```

Comment:
I think it should be better to move this static method to an utility class like XMLPositionUtility